SR2025/2024-09-04-confirm-early-places.md

```diff
@@ -0,0 +1,66 @@
+---
+to: Student Robotics 2024
+subject: Confirming your place at Student Robotics 2024
+attachments:
+  - Kit Disclaimer (PDF): https://drive.google.com/file/d/1dZAawbzQwh0peaZRpesL9Up5CzOr--Ec/view
+---
+
+Hi,
+
+You're in! We're happy to confirm you've got a place at Student Robotics 2025!
+
+Thank you for taking on the role of a [team supervisor][team-supervisor] for
+your team. If you have any questions about the [competition programme][programme-structure],
+building a robot or anything else, please feel free to reach out to us in the
+`#team-supervisors` channel on [Discord][discord] or email us at
+<teams@studentrobotics.org>.
+
+This year we're hosting [Kickstart][kickstart] at the University of Southampton,
+on Saturday 12th October. We will also be live-streaming the event if you are
+unable to attend in person. In that case we encourage that your team meet up to
+watch the presentations, get to know each other and start working together on
+the kit.
+
+As usual we will loan your team a [kit][kit] of parts to help build your robot.
+This year we hope to send out kits shortly before Kickstart, in early October.
+
+In order to enable us to send out your kit you need to agree to our _kit
+disclaimer_ (PDF attached). This disclaimer covers the terms and conditions
+under which you may use the kit and outlines the limits of our responsibilities.
+Please **sign it, scan it and email it back to us**.
+
+We will be sending your kit to the shipping address you provided when you signed
+up:
+
+    SHIPPING_ADDRESS
```

Comment:
We should also mention that we'll tell them when we send it and that will include a tracking link.

Reply:
ff46aec